ppr-ui/src/components/mhrRegistration/YourHome/ManufacturerMakeModel.vue

```diff
@@ -219,6 +227,13 @@ export default defineComponent({
       )
     )
 
+    const manufacturerNameRules = computed((): Array<Function> =>
+      customRules(
+        required("Enter a manufacturer's name or enter \"Unknown\" "),
+        maxLength(65)
+      )
+    )
+
```

Comment:
I don't mind the shorthand approach for function expressions (ie arrow function with no parameters{})!
Just keep in mind, most other components include computed values in the `localState` as well as parameters and explicit return statements.
ie `someValue: computed((): boolean => { return true })`